src/openms_gui/source/VISUAL/TableView.cpp

```diff
@@ -113,15 +118,36 @@ namespace OpenMS
     QStringList str_list;
 
     // write header
-    ts << getHeaderNames(WidgetHeader::VISIBLE_ONLY, true).join("\t") + "\n";
+    QStringList all_header_names = getHeaderNames(WidgetHeader::WITH_INVISIBLE, true);
```

Comment:
a bit too much boilerplate.
Could this be done in three lines like:
```
auto cols_to_export = (getHeaderNames(WidgetHeader::VISIBLE_ONLY, true) + mandatory_export_columns_);
cols_to_export.remove_duplicates();
```

Reply:
a nice didn't know that you can merge these lists that easily